sandbox/public/index.html

```diff
@@ -64,10 +64,10 @@
             doesOptInApply: getUrlParameter("legacyOptIn") === "true"
           });
           // Alloy only looks for window.Visitor when it initially loads, so only load Alloy after Visitor loaded.
-          includeScript("alloy.js");
+          includeScript("/alloy.js");
```

Comment:
We now have views that are multiple levels deep (`/personalization/products`), so we need to use an absolute path to reference alloy.js correctly in those instances.